pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -71,3 +102,230 @@ func (r *PerformanceProfileReconciler) hypershiftSetupWithManager(mgr ctrl.Manag
 
 	return ctrl.NewControllerManagedBy(mgr).For(&corev1.ConfigMap{}, builder.WithPredicates(p)).Complete(r)
 }
+
+func (r *PerformanceProfileReconciler) hypershiftReconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	if !ntconfig.InHyperShift() {
+		return reconcile.Result{}, fmt.Errorf("Using hypershift controller configuration while not in hypershift deployment")
+	}
```

Comment:
not blocking: this method is called only if the very same guard passed, do we need another check here?

Reply:
deleted